config/evss/mock_letters_response.yml.example

```diff
@@ -1,6 +1,6 @@
 ---
 # Mark Webb
-'796066621':
+'796104437':
```

Comment:
I was going to close this PR since I [merged the attribute nesting](https://github.com/department-of-veterans-affairs/vets-api/commit/6726b5d2cf9e7b090a0253c731fe038f439883ad) change this morning - but I hesitate to close based on this change.  Do mock user SSN's need to change?